mail/postfix/Makefile

```diff
@@ -116,6 +136,16 @@ ifdef CONFIG_POSTFIX_SQLITE
   AUXLIBS+=-L$(STAGING_DIR)/usr/lib -lsqlite3 -lpthread
 endif
 
+ifdef CONFIG_POSTFIX_PGSQL
+  CCARGS+=-DHAS_PGSQL -I$(STAGING_DIR)/usr/include/
+  AUXLIBS+=-L$(STAGING_DIR)/usr/lib -lpq
+endif
+
+ifdef CONFIG_POSTFIX_PCRE
+  CCARGS+=-DHAS_PCRE -I$(STAGING_DIR)/usr/include/
```

Comment:
for sure adding -DHAS_PCRE adds ~100kb to the compressed ipk file, didn't had time to test for functionality on PCRE, but I was asked to explicit add PCRE possibility, so it seems that before it didn't worked